cmd/kogito/command/install/data_index.go

```diff
@@ -231,12 +237,9 @@ func (i *installDataIndexCommand) Exec(cmd *cobra.Command, args []string) error
 		},
 	}
 
-	if err := kubernetes.ResourceC(i.Client).Create(&kogitoDataIndex); err != nil {
-		return fmt.Errorf(message.DataIndexErrCreating, err)
-	}
-
-	log.Infof(message.DataIndexSuccessfulInstalled, i.flags.Project)
-	log.Infof(message.DataIndexCheckStatus, kogitoDataIndex.Name, i.flags.Project)
-
-	return nil
+	return shared.
+		ServicesInstallationBuilder(i.Client, i.flags.Project).
+		OperatorInstalled().
```

Comment:
I would change this name to something like `SetOperatorInstalledOption(installed bool)` to be more readable

Reply:
I agree that we can change the name, maybe `OperatorAlreadyInstalled()`. But I won't add a  parameter, this makes the code more complicated than needed.

Reply:
ok for `OperatorAlreadyInstalled()`